lib/disk_based_memory.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
+require 'json'
+
 class DiskBasedMemory
   def state
     running_dir = File.dirname(__FILE__)
```

Comment:
You can use `__dir__` to get the directory of the current file.

Reply:
As in do you mean:
```ruby
  running_dir = File.dirame(__dir__)
```
?

Or just use __dir__ instead of our running_dir directory?

Reply:
`running_dir = __dir__`

Reply:
Either of those changes cause like half of our tests to fail.

Reply:
Ah that has fixed it. Thanks Craig 👍